.travis/run-linters.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-docker pull hadolint/hadolint
+docker pull hadolint/hadolint tag: v1.3.0
```

Comment:
CI fails :D https://travis-ci.org/wmde/wikibase-docker/jobs/421547431

I think this just needs to be vendor/imagename:tag